src/model/mpackage.nit

```diff
@@ -177,6 +178,11 @@ class MGroup
 end
 
 redef class Model
+
+	# Full package importation hierarchy
+	# Each package is in relation with itself
+	var mpackage_importation_hierarchy = new ReflexiveHashDigraph[MPackage]
```

Comment:
its more a graph than a hierarchy : no systematic transitivity and no anti-symmetry.

Reply:
I see. I changed the name for `mpackage_importation_graph`